openhands/core/schema/action.py

```diff
@@ -38,6 +38,10 @@ class ActionType(str, Enum):
     """Interact with the browser instance.
     """
 
+    MCP = 'call_tool_mcp'
```

Comment:
nit, maybe just MCP?

```suggestion
    MCP = 'mcp'
```

But we can always refactor later! No need to do it in this PR if it is too complex

Reply:
It was previously 'mcp', but I felt like it will be hard searching for the word 'mcp' in the workspace, so I changed it to call_tool_mcp so we can easily search and debug

Reply:
@enyst I replied the nit here